### PR TITLE
Fee Distribution major update

### PR DESCRIPTION
**Pull request by benoitdevos**
* Companion of https://github.com/logion-network/logion-pallets/pull/42
* Remove pallet node-authorization.
* Use `RewardDistributor` for inclusion fees.
* Group and set values for distribution keys.

logion-network/logion-internal#1049

### CODE REVIEW

File: runtime/PalletIds.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+| Pallet   | Name              | ID       | Account ID                                       | Comment      |
+|----------|-------------------|----------|--------------------------------------------------|--------------|
+| treasury | Treasury          | py/trsry | 5EYCAe5ijiYfyeZ2JJCGq56LmPyNRAKzpG4QkoQkkQNB5e6Z | No more used |
+| treasury | LogionTreasury    | py/lgtrs | ?                                                |              |
+| treasury | CommunityTreasury | py/cmtrs | ?                                                |              |
```
Comment (gdethier) [inline runtime/PalletIds.md:5]: `py/` is for Parity, we could use another prefix. Will the account IDs for Logion and community treasuries be set later?

File: runtime/src/lib.rs [hunk 2]
```diff
@@ -291,10 +291,72 @@ impl pallet_balances::Config for Runtime {
 }
 
 parameter_types! {
-    pub const TreasuryPalletId: PalletId = PalletId(*b"py/trsry");
-    pub TreasuryAccountId: AccountId = TreasuryPalletId::get().into_account_truncating();
-	pub const InclusionFeesToBurnPercent: u32 = 100;
-	pub const InclusionFeesTreasuryPercent: u32 = 0; // Inclusion fees disabled for the moment
+	pub const InclusionFeesDistributionKey: DistributionKey = DistributionKey {
```
Comment (gdethier) [inline runtime/src/lib.rs:294]: There is some indent issue here, probably due to a mix of tab/spaces.

File: runtime/src/lib.rs [hunk 3]
```diff
@@ -291,10 +291,72 @@ impl pallet_balances::Config for Runtime {
...
+        logion_treasury_percent: Percent::from_percent(0),
+        loc_owner_percent: Percent::from_percent(100),
+    };
+}
+
+parameter_types! {
+    pub const LogionTreasuryPalletId: PalletId = PalletId(*b"py/lgtrs");
+    pub LogionTreasuryAccountId: AccountId = LogionTreasuryPalletId::get().into_account_truncating();
+    pub const CommunityTreasuryPalletId: PalletId = PalletId(*b"py/cmtrs");
+    pub CommunityTreasuryAccountId: AccountId = CommunityTreasuryPalletId::get().into_account_truncating();
```
Comment (gdethier) [inline runtime/src/lib.rs:359]: Same remark as above w.r.t. the pallet ID string prefix.

**Review by gdethier — APPROVED** [review]
Good job! Updating dev JSON spec files implies that we will need to reset the network if we want to add a node, which is not a big issue but just wanted to make sure you were aware.

File: runtime/PalletIds.md (hunk 1 above)
Comment (benoitdevos) [reply runtime/PalletIds.md:5]: Yes I will set those ID's later on.

**Pull request merged**